net/libreswan/patches/010-use_dev_urandom.patch

```diff
@@ -0,0 +1,22 @@
+--- a/programs/pluto/plutomain.c
++++ b/programs/pluto/plutomain.c
+@@ -378,7 +378,7 @@ static void get_bsi_random(size_t nbytes
+ 	size_t ndone;
+ 	int dev;
+ 	ssize_t got;
+-	const char *device = "/dev/random";
++	const char *device = "/dev/urandom";
+ 
+ 	dev = open(device, 0);
+ 	if (dev < 0) {
+--- a/programs/rsasigkey/rsasigkey.c
++++ b/programs/rsasigkey/rsasigkey.c
+@@ -70,7 +70,7 @@
+ #define MIN_KEYBIT 2192
+ 
+ #ifndef DEVICE
```

Comment:
Possible to get around this by passing -DDEVICE="/dev/urandom" to CFLAGS I would think. Maybe there's a configure argument as well. No clue.

Reply:
seems to work ok without this on x86, I'll try on a mips machine

Reply:
tested on ubiquiti erx and is also working without the patch